easybuild/easyconfigs/h/h5py/h5py-2.9.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'h5py'
+version = '2.9.0'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
Why not multi_deps?

Reply:
Because there are python scripts in .../bin that gets hardcoded with the last python version in the multi_deps list and that is a no no...

Reply:
@akesandgren The only scripts i ended up with are
f2py     f2py2    f2py2.7  f2py3    f2py3.7
These are the scripts as in SciPy-bundle. This seems to be just garbage scripts that get installed with everything(?) that depends on SciPy-bundle. I find them in ASE, matplotlib for example.

Reply:
Ok, somewhat odd... perhaps we should figure out why and make sure the only exist in the SciPy bundle bin.

Reply:
They comes from numpy btw. And they are Fortran to Python interface generators.

Reply:
But in that case we can at least start with multi_dep:ing h5py .... give me a minute.

Reply:
`f2py` being copied is an artifact of using `setup.py install`, it won't happen when you're using `pip`

Reply:
Which we can't (use pip that is) due to the need for doing configure and build.